sonar-project.properties

```diff
@@ -20,15 +20,15 @@ sonar.eslint.reportPaths=packages/analytics-js/reports/eslint.json,packages/anal
 # Path to sources
 sonar.sources=packages
 sonar.inclusions=**/src/**/*.js, **/src/**/*.ts
-sonar.exclusions=**/*.json,**/*.html,**/*.png,**/*.jpg,**/*.gif,**/*.svg,**/nativeSdkLoader.js,assets/**/*,packages/analytics-js-plugins/src/errorReporting/event/**/*
```

Comment:
Why did you remove this line?
Whatever you need to add, add it at the end.

Reply:
Yes @MoumitaM I'll change it.